plugin.video.101barz/addon.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon
+	id="plugin.video.101barz"
+	name="101barz"
+	version="1.0.0"
+	provider-name="Skipmode A1">
+    <requires>
+        <import addon="xbmc.python"                     version="2.25.0"/>
```

Comment:
> WARN: For leia it is advised to set xbmc.python version to 2.26.0

Reply:
I set the version to 2.26.0, but now i get 'addon is not compatible with Kodi' without any more info what's wrong with it:
05:05:41.773 T:6584   DEBUG: ------ Window Init (DialogAddonInfo.xml) ------
05:05:41.773 T:6584    INFO: Loading skin file: DialogAddonInfo.xml, load type: KEEP_IN_MEMORY
05:05:41.779 T:6584   DEBUG: CAddonSettings[plugin.video.101barz]: loading setting definitions
05:05:41.779 T:6584   DEBUG: CAddonSettings[plugin.video.101barz]: trying to load setting definitions from old format...
05:05:41.780 T:6584   DEBUG: CAddonSettings[plugin.video.101barz]: loading setting values
05:05:42.806 T:6584    INFO: XCURL::DllLibCurlGlobal::CheckIdle - Closing session to http://mirrors.kodi.tv (easy=000001937376CF30, multi=0000019372E55AD0)
05:05:44.656 T:6584   DEBUG: CInputManager::ProcessMouse: trying mouse action leftclick
05:05:44.657 T:6584   ERROR: Add-on 'plugin.video.101barz' is not compatible with Kodi
05:05:44.659 T:6584   DEBUG: ------ Window Init (DialogNotification.xml) ------
05:05:50.380 T:6584   DEBUG: ------ Window Deinit (DialogNotification.xml) ------
05:05:55.581 T:6584   DEBUG: CInputManager::ProcessMouse: trying mouse action leftclick
05:05:55.581 T:6584   ERROR: Add-on 'plugin.video.101barz' is not compatible with Kodi
05:05:55.584 T:6584   DEBUG: ------ Window Init (DialogNotification.xml) ------
05:06:01.305 T:6584   DEBUG: ------ Window Deinit (DialogNotification.xml) ------
05:06:04.055 T:6584   DEBUG: CInputManager::ProcessMouse: trying mouse action leftclick

Please advise...

Reply:
Are you already using Kodi v18?

Reply:
yeah, when starting it says migrating. And then it says the addon is not compatible.

Reply:
Here it works, this looks more like a Kodi issue.

Reply:
Likely the upgrade didn't detect the bumped version of xbmc.python

Reply:
I found out what was wrong, i was using an older version of kodi 18. It works fine with the latest kodi 18.